backend/app/data_migrations/populate_geo_statistics.py

```diff
@@ -0,0 +1,46 @@
+import csv
+from typing import Union
+from app.db.models import (
+    Geography,
+    GeoStatistics
+)
+from app.db.session import SessionLocal
+
+
+def to_float(value: str) -> Union[float, None]:
```

Comment:
maybe we should add handling for garbage data here (rather than crashing when we fail to parse a float)?

Reply:
Ooo good point :+1:

Reply:
Also added unit tests